src/commands/@commands.yml

```
@@ -0,0 +1,99 @@
+checkout-to-workspace:
+  description: >
+    Check out source code and persist to workspace.
+  steps:
+    - checkout
+    - persist_to_workspace:
+        root: /root
+        paths:
+          - project
+set-version:
+  description: >
+    Set version for Sentry release, write to file and persist to workspace.
+
+    Will first run `git describe --tags` to set the version. If this fails, the output of
+    `sentry-cli releases propose-version` will be used as version.
+  steps:
+    - attach_workspace:
+        at: /root
+    - run:
+        name: Set version for Sentry release
+        command: |
+          if [ "$(git describe --tags 2>/dev/null)" ]; then
```

What is this check supposed to do?
```suggestion
          if [ -n "$(git describe --tags 2>/dev/null)" ]; then
```
Maybe also even simpler:
```suggestion
          if git describe --tags > /dev/null 2>&1; then
```